Dockerfile

```diff
@@ -1,11 +1,26 @@
 # Running from node
 from node:lts-alpine3.11
 
-run apk add python make g++ ffmpeg
+# Set time zone - default of America/Phoenix
+ARG TZ='America/Phoenix'
+ENV DEFAULT_TZ ${TZ}
+RUN apk add -U tzdata \
+	&& cp /usr/share/zoneinfo/${DEFAULT_TZ} /etc/localtime \
+	&& apk del tzdata
 
-COPY ./ ./
+# Install build depenacies
+RUN apk add make gcc g++
+
+# Install run depenacies
+RUN apk add python ffmpeg
```

Comment:
What's this for?

Reply:
They are for node-opus which is what is used to allow the discord.js to play audio over the voice channel. They already existed in a prior commit I just split them and cleaned them up to keep the docker image small